crates/net/network/src/transactions.rs

```diff
@@ -253,6 +253,7 @@ where
                 }
             }
             let mut new_pooled_hashes = hashes.build();
+            let new_full_transactions = full_transactions.build();
```

Comment:
this can now be moved into the else branch because we only need this for full txs